docs/theory/enzyme_kinetics.rst

```diff
@@ -3,3 +3,143 @@ Enzyme kinetics
 ===============
 
 This document explains the assumptions about enzyme kinetics that Maud uses.
+
+Modular rate law
+================
+
+This section outlines the assumptions made with the modular rate law, and includes a derivation of a 2 substrate (A, B),
+2 product (P, Q) random mechanism with competitive inhibitor I. It also highlights the general structure of the
+modular rate law used in Maud. The modular rate law framework was taken from [1],
+and was adapted to suit our structure. The general rate structure for the modular rate
+law is given below
+
+.. math::
+    v =  E_t f \frac{kcat_{1}\prod_{i, substrate}
```

Comment:
Maybe split this up - from the numerator it looks kind of like a full definition, so when I found the expressions `D` and `D^{reg}`in the denominator I did a bit of a double take.

You could maybe do it all in expressions and then explain them underneath?

Reply:
I've fixed this up now